requirements-direct.txt

```diff
@@ -0,0 +1,10 @@
+PyYAML
```

Comment:
Was a suggestion to place direct dependencies of drydock in requirements-direct.txt. Then after installing drydock w/ all dependencies, use pip freeze to get a list of all dependencies and versions for requirements.txt. This was a downstream dependency change doesn't possibly break something.